README.md

```diff
@@ -1,46 +1,37 @@
-# Getting Started with Create React App
+<h1 align="center">Space Burger</h1>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<div align="center">Web application for ordering in a burger shop using the burger constructor</div>
 
-## Available Scripts
+![Tech logos](https://i.ibb.co/DVFj8PL/tech-icons.jpg)
 
-In the project directory, you can run:
+![App screenshot](./readme_src/screenshot-app.PNG)
 
-### `yarn start`
+## What is this web application for? 🤷‍♀️
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+It is an interactive application that allows the user to conveniently assemble their own special burger using the constructor.
```

Comment:
It is an interactive application that allows users to assemble their special burgers using the constructor conveniently.